tests/integration/itest

```diff
@@ -30,7 +30,7 @@ def dockerized_dbs():
 
 
```

Comment:
These changes make sure that we have a GCS scratch bucket set up while running relevant integration tests manually.